server/routes/api/article.js

```diff
@@ -12,14 +12,23 @@ import BookmarkController from '../../controllers/BookmarkController';
 
 import RatingsController from '../../controllers/RatingsController';
 import RatingsValidation from '../../middlewares/validations/RatingsValidation';
+import ReportsController from '../../controllers/ReportsController';
+import ReportValidation from '../../middlewares/validations/ReportValidation';
 
 const router = express.Router();
 
 // Article endpoints
 router.get('/', ArticlesController.getArticles);
 router.get('/:slug', ArticlesController.getSingleArticle);
 
-// Ratting endpoint
+// Comment endpoints
+router.get(
+  '/:slug/comments/',
+  Queryvalidation.queryValidation,
```

Comment:
This should be `QueryValidation.queryValidation` it is a class name

Reply:
Thanks for pointing that out.